exporter/elasticsearchexporter/README.md

```diff
@@ -188,6 +191,7 @@ Documents may be optionally passed through an [Elasticsearch Ingest pipeline] pr
 This can be configured through the following settings:
 
 - `pipeline` (optional): ID of an [Elasticsearch Ingest pipeline] used for processing documents published by the exporter.
+- If `elasticsearch.document_pipeline` exists and is not an empty string in the log record attributes, then that pipeline will be used for that log record.
```

Comment:
Shouldn't this mention that this works when the `logs_dynamic_pipeline::enabled` property is set to `true`?